code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -316,6 +326,11 @@
 			if(udder && prob(5))
 				udder.add_reagent(milk_type, rand(5, 10))
 
+	if(has_toxingland)
+		if(stat == CONSCIOUS)
+			if(toxingland &&prob(2))
+				toxingland.add_reagent(toxin_type, rand(1, 2))
```

Comment:
There's no need for so many intentations, make it a single line, or two at most if you want the probability check to stand out

Reply:
I just copy-pasted the code for milking cows LOL

Reply:
Turns out the indentations have a purpose, if I remove the indentations it breaks the randomizer and probability code and carps produce toxin way too fast.